src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logoLink"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link nav_linki"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
```

Comment:
All navigation links have the same href attribute pointing to 'index.html'. This might be intentional for a single-page application, but if these are meant to navigate to different sections or pages, the href attributes should be updated accordingly.